File: TaskManager.Model/TaskInfo.cs
using System;
using TaskManager.Core;

namespace TaskManager.Model
{
    public class TaskInfo : ITaskInfo
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public int Priority { get; set; }
        public DateTime TimeToComplete { get; set; }
        public DateTime Added { get; set; }
    }
}

File: TaskManager.Client/src/app/models/TaskInfo.ts
export class TaskInfo {
  constructor(
    public name: string,
    public description: string,
    public priority: number,
    public timeToComplete: TimeSpan,
    public featureId: number
  ) { }
}

export class TimeSpan {
  constructor(
    public weeks: number,
    public days: number,
    public hours: number
  ) { }
}
